mm2src/coins/nft.rs

```diff
@@ -745,11 +811,91 @@ where
         let tx_meta = TxMeta {
             token_address: nft_meta.token_address,
             token_id: nft_meta.token_id,
+            token_uri: nft_meta.token_uri,
             collection_name: nft_meta.collection_name,
-            image: nft_meta.uri_meta.image,
+            image_url: nft_meta.uri_meta.image_url,
             token_name: nft_meta.uri_meta.token_name,
         };
         storage.update_txs_meta_by_token_addr_id(chain, tx_meta).await?;
     }
     Ok(())
 }
+
+/// `contains_disallowed_scheme` function checks if the text contains some link.
+fn contains_disallowed_scheme(text: &str) -> bool {
+    text.contains("http://") || text.contains("https://") || text.contains("ftp://") || text.contains("file://")
```

Comment:
Do we allow all other schemes? https://en.wikipedia.org/wiki/List_of_URI_schemes

Reply:
I have added a filter for more likely schemes, well I believe we can leave just `text.contains("://")`

Reply:
If we are worried about the urls not just the schemes, you can consider doing it with regex

Reply:
coins Cargo.toml doesnt have this dependency, I believe we can filter any text which can contain any scheme. it should be just text.

Reply:
Done, added regex usage and some more tests for spam check